src-tauri/tauri.conf.json

```diff
@@ -48,7 +48,7 @@
       "endpoints": [
         "https://tauri-updater.subspace.network/v1/{{target}}/{{arch}}/{{current_version}}"
       ],
-      "dialog": true,
+      "dialog": false,
```

Comment:
hmm why don't we need to enable dialog's ?

Reply:
has to be disabled in order for events to work - https://tauri.app/v1/guides/distribution/updater#events